Dockerfile

```diff
@@ -13,26 +9,41 @@ RUN apt-get update \
            libqtwebkit-dev \
            postgresql-client
 
+RUN useradd --uid 2003 --home /opt/homs --shell /bin/bash --comment "HOMS" homs
```

Comment:
This UID reserved for imprint application: https://wiki.hydra-billing.ru/pages/viewpage.action?pageId=42699319